lib/doc2pdf.rb

```diff
@@ -31,10 +33,10 @@ def self.replace!(document:, replacer:)
   def self.replace_and_save!(document:, output_base_path:, replacer:)
     FileUtils.mkdir_p(File.dirname(output_base_path))
 
-    replace!(document: document, replacer: replacer)
+    document = replace!(document: document, replacer: replacer)
 
-    doc_path = "#{output_base_path}.doc"
-    pdf_path = "#{output_base_path}.pdf"
+    doc_path = "#{output_base_path}/file.doc"
```

Comment:
This is an arbitrary convention that I think could pose a tricky constraint to the users.
Not to mention that if we want to do so, then we have to update the documentation accordingly.

Reply:
It was throwing error without name, gonna add a filename param

Reply:
I added the instructions in the `README.md`:

<img width="389" alt="Screenshot 2021-04-10 at 18 23 09" src="https://user-images.githubusercontent.com/21038/114277059-d7376300-9a29-11eb-979e-33726fb9e9ea.png">

RTFM :-D

Reply:
@xkraty what about this? 😅